package.json

```diff
@@ -76,12 +76,14 @@
     "test:e2e": "yarn workspace @okta/e2e test",
     "test:e2e:lang": "yarn workspace @okta/e2e test:lang",
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start --mock mockDuo --open false",
-    "test:testcafe-run": "yarn test -t testcafe -c 4 chrome:headless --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
+    "test:testcafe-run": "yarn test -t testcafe --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
```

Comment:
Moved concurrency and browser settings from CLI to `.testcaferc.js` to be able to configure them depending on environment